frontend/test/__support__/e2e/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "target": "esnext",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "types": ["cypress", "@testing-library/cypress"],
+    "moduleResolution": "node",
+    "allowJs": true,
+    "paths": {
+      "*": ["../../*", "../../../src/*"]
```

Comment:
I've only encountered that we import some files from our main app in Cypress-related files in some places, but if we import from any other places that are defined in our main `tsconfig.json` then I'll happily add those, but I just want to keep these configs as concise as possible initially.

https://github.com/metabase/metabase/blob/07c91cd608860798442667074979f514a317b182/tsconfig.json#L26-L35